app/controllers/receipts_controller.rb

```diff
@@ -0,0 +1,28 @@
+class ReceiptsController < ApplicationController
+    def new
+    end
+    
+    def receipts
+        if (params[:user_id])
```

Comment:
user = current_user works better here because you can eliminate the next line of code as well.